hack/make-rules/vet.sh

```diff
@@ -23,9 +23,16 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 cd "${KUBE_ROOT}"
 
+# If called directly, exit.
+if [ "${CALLED_FROM_MAIN_MAKEFILE:-""}" == "" ]; then
```

Comment:
nit: I think you might need to use `[[ ]]` for `==` to work? we generally prefer `[[ ]]` anyway.